src/style.css

```diff
@@ -1 +1,69 @@
-/* Styles go here */
+html {
+  font-family: "Roboto", Helvetica, sans-serif;
+  font-weight: 500;
+}
+
+body {
+  margin: 0;
+}
+
+.header {
+  max-width: 1200px;
+  min-width: 1024px;
+  padding: 0 50px;
+  height: 60px;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  font-size: 12px;
+}
+
+.logo {
```

Comment:
add `display:block` to an image, so link will be the same 40x40
<img width="193" alt="Screenshot 2023-04-10 at 21 03 07" src="https://user-images.githubusercontent.com/117587463/230963361-b9266e91-b8dc-4824-9428-0093b28555fb.png">